### ISSUE TITLE
Godot cant find DLL on windows

### ISSUE THREAD

**Issue by CypherDoesStuff** [tcmug/godot_trailmesh#8]
Using Godot 4.3 RC 2

Seems like using the addon, Godot gives errors that it cant find the right DLL since its looking for the wrong file path.
```  
platform/windows/os_windows.cpp:383 - Condition "!FileAccess::exists(path)" is true. Returning: ERR_FILE_NOT_FOUND
  GDExtension dynamic library not found: D:/Godot/ProjectRandom/addons/trailmesh/bin/windows/libtrailmesh.windows.template_debug.x86_64.dll
Failed loading resource: res://addons/trailmesh/trailmesh.gdextension. Make sure resources have been imported by opening the project in the editor at least once.
  ```

**Comment by Tshmofen** [comment]
The same thing on 4.2.2. Though, worked it around by just renaming all libraries in `trailmesh\bin\windows\` to the same name with `lib` prefix: `trailmesh.windows.template_debug.x86_32.dll` -> `libtrailmesh.windows.template_debug.x86_32.dll`. The trails seems to be working as expected.

**Comment by tcmug** [comment]
This must be a bug inherited from [the template](https://github.com/godotengine/godot-cpp-template/blob/main/demo/bin/example.gdextension).

This should also be fixable by changing the paths to the correct ones in [project/addons/trailmesh/trailmesh.gdextension](https://github.com/tcmug/godot_trailmesh/blob/main/project/addons/trailmesh/trailmesh.gdextension)

Will fix this soonest.

**Comment by tcmug** [comment]
Fixed in version 0.51
